### ISSUE TITLE
`gridOptions.allowShowChangeAfterFilter` not recognised as a valid property

### ISSUE THREAD

**Issue by mfreeborn** [ag-grid/ag-grid#5682]
I've taken a plunker example from the docs and added a single additoinal`gridOptions` parameter: `allowShowChangeAfterFilter`:

https://plnkr.co/edit/qANiOdZqXSf8G8eA?open=main.js

In the console, there is an error reporting that the above is an invalid property. However, it is defined [here](https://www.ag-grid.com/javascript-data-grid/grid-options/#reference-rendering-allowShowChangeAfterFilter) in the docs.

Should this not be a valid property?

Using version 28.2.0.

**Comment by StephenCooper** [comment]
Hello,

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference and description:
AG-7396 allowShowChangeAfterFilter is missing from PropertyKeys

It's a small fix to our validation logic so will be in our next release. 

Now that this issue is recorded in our backlog, we will soon close the GitHub issue because we will not update it when this bug is fixed in a future version of AG Grid. The best way to get an update on this item is to sign up for AG Grid new release notifications. Once a new AG Grid version is released, look for the item reference above (AG-7396) on the changelog page to see if it's resolved in that version:
https://www.ag-grid.com/ag-grid-changelog/

Thanks again for bringing this up with us - we appreciate your time!
